README.md

```diff
@@ -1,114 +1,112 @@
-# Welcome to the QuestCraft GitHub!
-### **[Support us on Patreon!](https://patreon.com/QuestCraftXR)**
-
-# QuestCraft
+ # Welcome to the QuestCraft GitHub!
 ![QC](/QCSimple3.jpg)
 
 QuestCraft, utilizing Vivecraft and Pojlib, is a standalone port of Minecraft: Java Edition to Oculus Quest Headsets. With a team of four main developers and 8 third-party developers, over the past year, we have not only gotten the bestselling game, Minecraft: Java Edition running on Android, but we have also worked with third-party developers to bring it into Virtual Reality on the Oculus Quest line of standalone HMDs.
 
+### **[Support us on Patreon!](https://patreon.com/QuestCraftXR)**
+
 # Features
 - ### **Roomscale Minecraft VR!** 
-- ### **Quest 2 Native!\***
+- ### **Quest 1/2/Pro/3 Native!\***
 - ### **Full Multiplayer!**                                                                    
 - ### **Immersive Controls through Vivecraft!**                                                                                                   
-- ### **Supports 1.18.2 (most stable version), 1.19.2-4, and 1.20.0-1!****                                                                                                                                 
-- ### **Includes a launcher that allows you to start QuestCraft, switch between versions and renderers, automatically update installed mods and other MC components in milliseconds, and use Mod Manager, a tool to download and install supported mods right from your Quest!**
-**\*Quest 1 Support is no longer available.**                                                                                                                                                                                       
+- ### **Supports 1.18.2 (most stable version as of now), 1.19.2/4, and 1.20.1-2/4!**                                                                                                                                 
```

Comment:
1.18.2 doesnt currently work and support for it will be dropped once vivecraft stops